### ISSUE TITLE
Validation metrics when doing CV

### ISSUE THREAD

**Issue by peroap** [ustunb/risk-slim#13]
Hi Berk, I am wondering if there is any way to get the performance metrics of the model when using the cv_indices flag (I am doing a 10 fold-cv). I am mainly interested in accuracy, balanced_acc, precision, recall, AUROC and AUPR of the folds.

I have written an independent script myself and get the overall performance metrics, but since I am using the same data for getting the coefficients vector and for the validation, the model is overfitting. I have noticed that the model outputs one coefficient vector for all of the folds, instead of one for each iteration.

Since you mention in your paper that you are doing nested CV, was wondering if you were saving the metrics for each fold somewhere.

Thanks for the help and congrats for the great library!

**Comment by ustunb** [comment]
Hi!

Some answers:

RE: Performance Metrics. I actually don't have code to compute performance metrics in the package. I just use the metrics from scikit-learn. 

RE:CV. I assume that you're training the model using [batch/job_template.sh.sh](https://github.com/ustunb/risk-slim/blob/100aca0fc0475e1ffb22f5ecf8ba240384a10adb/batch/job_template.sh)" (since its the only file that has a `cvindices` plage). I think that what might be happening is that you are calling this script once assuming that it will do the 10-CV for you. In reality, the script will only train one of the models you need for a 10-CV – and the `cvindices` and `fold` parameters are used to specify the models you need.

To run a 10-CV you should call this script 11 times, and change the value of the `fold` parameter [on this line](https://github.com/ustunb/risk-slim/blob/100aca0fc0475e1ffb22f5ecf8ba240384a10adb/batch/job_template.sh#L46). Specifically:

1. You  will want to run the script setting `fold = 0`. This will train a model using the entire dataset. This is the model that you will want to deploy.

2. You will want to run the script setting `fold = 1, 2... 10`. This will using 90% of the dataset. You can then use the metrics from these models to compute out-of-sample accuracy metrics.

Hope this helps!
